adafruit_esp32spi/adafruit_esp32spi_socketpool.py

```diff
@@ -60,12 +56,12 @@ def getaddrinfo(  # pylint: disable=too-many-arguments,unused-argument
         if not isinstance(port, int):
             raise ValueError("Port must be an integer")
         ipaddr = self._interface.get_host_by_name(host)
-        return [(SocketPoolContants.AF_INET, socktype, proto, "", (ipaddr, port))]
+        return [(SocketPool.AF_INET, socktype, proto, "", (ipaddr, port))]
```

Comment:
This should not be a qualified name now.
```suggestion
        return [(AF_INET, socktype, proto, "", (ipaddr, port))]
```

Reply:
@dhalbert happy to add this and the pylint ignore, but thought this was cleaner (and it works):
<img width="804" alt="image" src="https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/assets/1547881/fd1920b9-a541-42db-bcf6-ddb4ed32a067">

Reply:
OK, I see, sorry. I was just testing by importing it into CPython, which did not complain. But it doesn't run.